    public Throwable getRootCauseException() {
        return rootCauseException;
    }

